Formula/joern.rb

```diff
@@ -0,0 +1,73 @@
+class Joern < Formula
+  desc "Open-source code analysis platform based on code property graphs"
+  homepage "https://joern.io/"
+  url "https://github.com/joernio/joern/archive/refs/tags/v1.1.1384.tar.gz"
+  sha256 "c71b77c519b051a07728e20eb78538f6c76287e7ec2ee4ee082a8fd852f67dad"
+  license "Apache-2.0"
+
+  depends_on "node@18" => :build
+  depends_on "sbt" => :build
+  depends_on "yarn" => :build
+  depends_on "coreutils"
+  depends_on "openjdk@11"
```

Comment:
Does it not work with the latest or at least 17?

Reply:
The installation guide for Joern states:

> JDK 11 (get here for example: https://openjdk.java.net/install/); newer JDK versions might work, but have not been properly tested.

Additionally, all the Github Workflows for the product use JDK 11. I felt it was safest to build with the recommended version so that users don't encounter potential unexpected errors. Should I add a comment indicating this to the formula?

Reply:
No, "not tested" is not a good reason to stay with a 4 year old Java version. We should use the latest version and report issues upstream.

Reply:
we've also just upgraded our default JDK to 17: https://github.com/joernio/joern/pull/2182

Reply:
Fixed as of https://github.com/Homebrew/homebrew-core/pull/120622/commits/b637d02b06d0ff043121d1b0ad51a91fc343de41